### PR TITLE
Remove reference to old resource Notifications which have not been used for quite a while, Notifiers have to be used

### PR DESCRIPTION
**Pull request by Soulou**

### CODE REVIEW

File: events_boilerplate.go [hunk 1]
```diff
@@ -96,15 +96,6 @@ func (e *EventEditVariableType) TypeDataPtr() interface{} {
 func (e *EventEditVariablesType) TypeDataPtr() interface{} {
 	return &e.TypeData
 }
-func (e *EventNewNotificationType) TypeDataPtr() interface{} {
```
Comment (EtienneM) [inline events_boilerplate.go]: Maybe the events can still be returned by the API and we should keep this? Or data have been migrated?

**Comment by EtienneM** [comment]
Don't forget to update the changelog please

**Comment by Soulou** [comment]
Data were migrated

**Comment by Soulou** [comment]
ok doing it

**Pull request merged**